docs/developing-locally.rst

```diff
@@ -26,6 +26,8 @@ First things first.
 #. Install development requirements: ::
 
     $ pip install -r requirements/local.txt
+    # Optional: install git hook if using pre-commit
+    $ pre-commit install
```

Comment:
Can we expand this section a bit? We could link to the pre-commit documentation for details.

Reply:
I will add a note which is a link referred to the usage of `pre-commit` in its docs under the `pip`, like the process of creating database:

```#. Create a new PostgreSQL database using createdb_: ::

    $ createdb <what you have entered as the project_slug at setup stage> -U postgres --password <password>

   .. note::

       if this is the first time a database is created on your machine you might need an
       `initial PostgreSQL set up`_ to allow local connections & set a password for
       the ``postgres`` user. The `postgres documentation`_ explains the syntax of the config file
       that you need to change.
```
The note I will add as follows:

```
    .. note::

        the `pre-commit` will exist in the generated project as default.
        so, for the usage of `pre-commit`, take this [link](https://pre-commit.com/).
```

what do you think?